react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Stopwatch extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {second: 0, timer: false};
+    this.play = this.play.bind(this);
+    this.pause = this.pause.bind(this);
+    this.reset = this.reset.bind(this);
+  };
+  play(){
+    this.intervalId = setInterval(() => {
+      this.setState({
+        second: this.state.second + 1,
+        timer: true
+      });
+    }, 1000);
+  };
+  pause(){
+    clearInterval(this.intervalId);
+    this.setState({timer: false});
+  };
+  reset() {
+    this.setState({second: 0,
+    timer: false
+  });
+  };
+  render() {
+    if (this.state.second === 0) {
```

Comment:
Approved, with the following note. It is best to keep the render method as clean as possible and reduce the amount of JSX which is present. What you have done is not wrong, but I recommend striving to make the decisions of what to render in another function.